src/backend/dx11/src/conv.rs

```diff
@@ -0,0 +1,82 @@
+use hal::format::{Format};
+
+use winapi::um::d3d11::*;
+use winapi::um::d3dcommon::*;
+use winapi::shared::dxgiformat::*;
+
+// TODO: stolen from d3d12 backend, maybe share function somehow?
```

Comment:
speaking of sharing, which of the things here are from `d3d11`?

Reply:
You mean which formats are compatible with dx11?

Reply:
I mean, what part of `winapi::um::d3d11` is used here?

Reply:
Ah, good catch 😄